### PR TITLE
35700 rename inset area to position area ff release

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added Firefox 131 release note for `position-area`

### Motivation

Working on #35700

### Related issues and pull requests

- [BCD PR](https://github.com/mdn/browser-compat-data/pull/24454)

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/131/index.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ This article provides information about the changes in Firefox 131 that affect d
 
 ### CSS
 
+- The `inset-area` CSS property has been renamed to {{CSSXRef("position-area")}}, this is part of the work for [Anchor Positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning), so at present this property does nothing without the {{CSSXRef("anchor-name")}} and {{CSSXRef("position-anchor")}} properties. ([Firefox bug 1909673](https://bugzil.la/1909358) and [Firefox bug 1909673](https://bugzil.la/1838746)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/131/index.md:21]:
```suggestion
- The `inset-area` CSS property has been renamed to {{CSSXRef("position-area")}}, this is part of the work for [Anchor Positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning), so at present this property does nothing without the {{CSSXRef("anchor-name")}} and {{CSSXRef("position-anchor")}} properties. ([Firefox bug 1909358](https://bugzil.la/1909358) and [Firefox bug 1838746](https://bugzil.la/1838746)).
```

Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/131/index.md:21]: I think this covers it, does that look right?

Comment (dletorey) [reply files/en-us/mozilla/firefox/releases/131/index.md]: much more eloquent than I wrote

**Review by bsmth — APPROVED** [review]
Nice, thanks!

**Pull request merged**